tools/logger/convert.c

```diff
@@ -849,19 +849,25 @@ int convert(struct convert_config *config)
 		      config->ldc_fd);
 	if (!count) {
 		log_err("failed to read uuid section data.\n");
-		return -ferror(config->ldc_fd);
+		ret = -ferror(config->ldc_fd);
```

Comment:
Sorry, not following the commit message. What is leaking and how are we fixing it.

Reply:
Changed commit message